frontend/packages/knative-plugin/src/utils/knatify-utils.ts

```diff
@@ -48,7 +49,15 @@ export const getKnatifyWorkloadData = (obj: K8sResourceKind) => {
       template: {
         metadata: {
           labels: templateMetadata?.labels ?? {},
-          annotations: templateMetadata?.annotations ?? {},
+          annotations: {
+            ...(templateMetadata?.annotations && templateMetadata.annotations),
+            ...(hpaSpec?.minReplicas && {
+              'autoscaling.knative.dev/minScale': `${hpaSpec.minReplicas}`,
```

Comment:
import and use const here
```suggestion
              KNATIVE_MINSCALE_ANNOTATION: `${hpaSpec.minReplicas}`,
```